### PR TITLE
[ONEM-19607] Auth module stub

### PR DESCRIPTION
**Pull request by mariusz-sikora-red**

### CODE REVIEW

File: LISA/AuthModule/Auth.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#ifndef AUTH_H
```
Comment (szymon-polke-red) [inline LISA/AuthModule/Auth.h:21]: #pragma once ?

File: LISA/Executor.cpp [hunk 2]
```diff
@@ -311,7 +313,11 @@ void Executor::doInstall(std::string type,
 {
     INFO("url=", url, " appName=", appName, " cat=", category);
 
-    // TODO check authentication method
+    auto authMethod = getAuthenticationMethod("", "", "");
```
Comment (szymon-polke-red) [inline LISA/Executor.cpp:316]: type.c_str(), id.c_str(), url.c_str() ?

File: LISA/Executor.cpp [hunk 3]
```diff
@@ -311,7 +313,11 @@ void Executor::doInstall(std::string type,
...
+    if(NONE != authMethod) {
```
Comment (szymon-polke-red) [inline LISA/Executor.cpp:317]: Shouldn't header file be included for enum visibility?

Comment (mariusz-sikora-red) [reply LISA/Executor.cpp:318]: It was lost after merging to latest changes.

**Review by szymon-polke-red — APPROVED** [review]

**Review by piotr-marcinkowski-red — APPROVED** [review]

**Pull request merged**